### PR TITLE
Office hours subpage

### PR DESCRIPTION
**Pull request by kpodemski**

### CODE REVIEW

File: content/office-hours/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Office hours
+
+menu:
+  main:
+    parent: 'Community'
+    name: 'Office hours'
+    weight: 100
+---
+
+# PrestaShop Project's Office hours
```
Comment (coffrep) [inline content/office-hours/_index.md:11]:
```suggestion
# PrestaShop Project's Office Hours
```

File: content/office-hours/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.
+
+These meetings have already proven it was a great idea. There have been constructive discussions about many ongoing projects, Pull Requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```
Comment (coffrep) [inline content/office-hours/_index.md:15]:
```suggestion
These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```

File: content/office-hours/_index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Join the next meeting!
+
+Would you like to discuss a Pull Request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
```
Comment (coffrep) [inline content/office-hours/_index.md:19]:
```suggestion
Would you like to discuss a pull request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
```

File: content/office-hours/_index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except days that are bank holidays in most European countries) from 11:00 AM to 12 PM CET.
```
Comment (coffrep) [inline content/office-hours/_index.md:21]:
```suggestion
Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except bank holiday days in most European countries) from 11:00 AM to 12 PM CET.
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hey - I've just added a few suggestions. I checked on the GitHub site and they don't capitalize the word pull request.

**Review by coffrep — APPROVED** [review]
All good thanks Kpodemski

**Review by nicosomb — APPROVED** [review]

File: content/office-hours/_index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+menu:
+  main:
+    parent: 'Community'
+    name: 'Office hours'
+---
+
+# PrestaShop Project's Office Hours
+
+The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.
```
Comment (eternoendless) [inline content/office-hours/_index.md:12]:
```suggestion
The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss everything related to the project.
```

File: content/office-hours/_index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```
Comment (eternoendless) [inline content/office-hours/_index.md:14]: I think it'd be best to keep the tone to a more "matter-of-fact", less opinionated style: what kinds of topics are discussed, why would it be interesting for me to participate?

File: content/office-hours/_index.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+## Join the next meeting!
+
+Would you like to discuss a pull request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
+
+Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except bank holiday days in most European countries) from 11:00 AM to 12 PM CET.
+
+If you want to suggest topics for discussion, please [fill out this form](https://forms.gle/2r7jPp5WxhFSUQme8) (it is not required, though!).
```
Comment (eternoendless) [inline content/office-hours/_index.md:22]:
```suggestion
If you want to suggest topics for discussion, please consider [filling out this form](https://forms.gle/2r7jPp5WxhFSUQme8).
```

File: content/office-hours/_index.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.
+
+These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
+
+## Join the next meeting!
+
+Would you like to discuss a pull request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
+
+Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except bank holiday days in most European countries) from 11:00 AM to 12 PM CET.
```
Comment (eternoendless) [inline content/office-hours/_index.md:20]:
```suggestion
Office Hours are held on [this Google Meet room](https://meet.google.com/fer-hcgk-wti) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), except for bank holiday days in most European countries.
```

File: content/office-hours/_index.md (hunk 6 above)
Comment (kpodemski) [reply content/office-hours/_index.md]: @eternoendless let me know if the new version is better

**Review by 0x346e3730 — APPROVED** [review]

File: content/office-hours/_index.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+menu:
+  main:
+    parent: 'Community'
+    name: 'Office hours'
+---
+
+# PrestaShop Project's Office Hours
+
+The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```
Comment (eternoendless) [inline content/office-hours/_index.md:12]:
```suggestion
The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer questions from the community and help contributors finish their contributions.
```

**Comment by eternoendless** [comment]
Thank you @kpodemski

**Pull request merged**